documentation/examples/prometheus-digitalocean.yml

```diff
@@ -0,0 +1,27 @@
+# A example scrape configuration for running Prometheus with
+# DigitalOcean.
+
+scrape_configs:
+  
+  # Make Prometheus scrape itself for metrics.
+  - job_name: 'prometheus'
+    static_configs:
+    - targets: ['localhost:9090']
+
+  # Discover Node Exporter instances to scrape.
+  - job_name: 'node'
+
+    digitalocean_sd_configs:
+      - bearer_token: 8034e306bdb44ad882096043353fb303ba58aeff384b43c1a36b8926cb00bf8a
```

Comment:
This looks suspiciously like an actual value. I'd make it more obvious that this needs to be substituted in.